### PR TITLE
pkg/mount/magic.go: Add BTRFS magic value

### PR DESCRIPTION
**Pull request by CodingVoid**
BTRFS partitions can now be mounted

source: https://btrfs.readthedocs.io/en/latest/dev/On-disk-format.html#superblock

### CODE REVIEW

File: pkg/mount/magic.go [hunk 1]
```diff
@@ -201,7 +201,7 @@ func FSFromBlock(n string) (fs string, flags uintptr, err error) {
 		return "", 0, err
 	}
 	defer f.Close()
-	block := make([]byte, blocksize)
+	block := make([]byte, blocksize*2)
```
Comment (rminnich) [inline pkg/mount/magic.go:204]: what is the reason for blocksize * 2? Will this break any other file systems? I really don't know, just wondering.

**Review by rminnich — APPROVED** [review]
I just had one question about the change. 
Thanks very much for your contribution!

File: pkg/mount/magic.go (hunk 1 above)
Comment (CodingVoid) [reply pkg/mount/magic.go:204]: The problem is that the BTRFS magic value is inside the superblock, which is exactly 64 bytes AFTER the first blocksize (offset 65536+64 aka 0x10040), so we need to read more than 1 blocksize to determine a BTRFS file system.

Comment (rminnich) [reply pkg/mount/magic.go:204]: This is a very clear explanation, I wonder if you might put it in as a comment? I had no idea.

**Review by rminnich — APPROVED** [review]

**Pull request merged**